destination/src/test/java/com/userleap/destination/SprigDestinationTests.kt

```diff
@@ -28,7 +28,16 @@ class SprigDestinationTests {
 
     @Test
     fun `settings are updated correctly`() {
-        every { Sprig.configure(any(), TEST_ENV, mapOf("x-ul-installation-method" to "android-segment")) } returns Unit
+        every { 
+            Sprig.configure(
+                any(),
+                TEST_ENV, 
+                mapOf(
+                    "x-ul-installation-method" to "android-segment",
+                    "x-ul-package-header" to any()
```

Comment:
@earlsprig we can't use `any()` like this, it will use value returned from `any()` as `"x-ul-package-header"`. It will not mock it.

We have two options here:
1) Use exact value:
```.kt
mapOf(
    "x-ul-installation-method" to "android-segment",
    "x-ul-package-version" to "1.2.2"
)
``` 
2) Use `any()` for the entire map:
```.kt
Sprig.configure(
  any(),
  TEST_ENV,
  any()
)
```

Reply:
UPD: we can use matchers as well:
```.kt
every {
    Sprig.configure(
        match { it is Application },
        TEST_ENV,
        match {
            it["x-ul-installation-method"] == "android-segment" &&
            it["x-ul-package-version"] != null
        }
    )
} returns Unit
```

Reply:
@vcherneny thank you!